python/hail/methods/statgen.py

```diff
@@ -1847,18 +1847,160 @@ def rrm(call_expr):
+    population allele frequencies by :math:`p_{k, m}`, and diploid, unphased
+    genotype calls by :math:`g_{n, m}` (0, 1, and 2 correspond to homozygous
+    reference, heterozygous, and homozygous variant, respectively).
+    
+    The generative model is then given by:
+
+    .. math::
+        k_n \,&\sim\, \pi
+
+        p_m \,&\sim\, P_0
+
+        p_{k,m} \mid p_m\,&\sim\, \mathrm{Beta}(\mu = p_m,\, \sigma^2 = F_k p_m (1 - p_m))
+
+        g_{n,m} \mid k_n, p_{k, m} \,&\sim\, \mathrm{Binomial}(2, p_{k_n, m})
+
+    The beta distribution by its mean and variance above; the usual parameters
+    are :math:`a = (1 - p) \frac{1 - F}{F}` and :math:`b = p \frac{1 - F}{F}` with
+    :math:`F = F_k` and :math:`p = p_m`.
+
+    The resulting dataset has the following fields.
+
+    Global fields:
+
+    - **num_populations** (*Int32*) -- Number of populations.
+    - **num_samples** (*Int32*) -- Number of samples.
+    - **num_variants** (*Int32*) -- Number of variants.
+    - **pop_dist** (*Array[Float64]*) -- Population distribution indexed by
+      population.
+    - **fst** (*Array[Float64]*) -- :math:`F_{ST}` values indexed by
+      population.
+    - **ancestral_af_dist** (*Struct*) -- Description of the ancestral allele
+      frequency distribution.
+    - **seed** (*Int32*) -- Random seed.
+
+    Row fields:
+
+    - **v** (*Variant*) - Variant (key field).
+    - **ancestral_af** (*Float64*) -- Ancestral allele frequency.
+    - **af** (*Array[Float64]*) -- Modern allele frequencies indexed by
+      population.
+
+    Column fields:
+
+    - **s** (*String*) - Sample ID (key field).
+    - **pop** (*Int32*) -- Population of sample.
+
+    Entry field:
+
+    - **GT** (*Call*) -- Genotype call (unphased, diploid).
+
+    Parameters
+    ----------
+    num_populations : :obj:`int`
+        Number of modern populations.
+    num_samples : :obj:`int`
+        Total number of samples.
+    num_variants : :obj:`int`
+        Number of variants.
+    num_partitions : :obj:`int`, optional
+        Number of partitions.
+        Default is 1 partition per million entries or 8, whichever is larger.
+    pop_dist : :obj:`list` of :obj:`float`, optional
+        Unnormalized population distribution, a list of length
+        ``num_populations`` with non-negative values.
+        Default is ``[1, ..., 1]``.
+    fst : :obj:`list` of :obj:`float`, optional
+        :math:`F_{ST}` values, a list of length ``num_populations`` with values
+        in (0, 1). Default is ``[0.1, ..., 0.1]``.
+    af_dist : :class:`.UniformDist` or :class:`.BetaDist` or :class:`.TruncatedBetaDist`
+        Ancestral allele frequency distribution.
+        Default is ``UniformDist(0.1, 0.9)``.
+    seed : :obj:`int`
+        Random seed.
+    reference_genome : :class:`.GenomeReference`, optional
+        The row key will have type `TVariant(reference_genome)`.
+        Reference genome to use. Default is :class:`~.HailContext.default_reference`.
```

Comment:
Can you change the order of these. Reference genome to use. Default is ... The row key will have type ... .